<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FControl</class>
 <widget class="QDialog" name="FControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Qualité</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_ajouter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_control_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupe_actes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Actes</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Qualité</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Echeances</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="echenaces_text">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Type du Produit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_qualite">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bonne</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mauvaise</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="combo_type_prod">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Electrique</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>FControl</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
